src/server/config/schema.js

```diff
@@ -29,6 +29,8 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
+  uuid: Joi.string().guid().default(),
```

Comment:
Initially @rashidkpc and I figured that having it at the top level of the
config would be appropriate.

But since the type is named 'server' in the .kibana index now, it does seem
like uuid should be nested in the server object.
On Wed, Apr 6, 2016 at 6:58 AM Jonathan Budzenski <notifications@github.com>
wrote:

> In src/server/config/schema.js
> <https://github.com/elastic/kibana/pull/6583#discussion_r58709237>:
>
> > @@ -29,6 +29,8 @@ module.exports = () => Joi.object({
> >      exclusive: Joi.boolean().default(false)
> >    }).default(),
> >
> > +  uuid: Joi.string().guid().default(),
>
> should this be nested in the server object?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/kibana/pull/6583/files/f03c45268075416b6ab543a0b420e9c2aafa49f9#r58709237>
>